src/action/bandaidset.rs

```diff
@@ -104,15 +103,11 @@ impl FirstAidKit {
 
             // process all subsequent lines
             while let Some(replacement) = replacement_lines.next() {
-                dbg!(&replacement);
-                let line = span_lines
-                    .next()
-                    // TODO: How can we get rid of lines? E.g., original content had 4 lines, replacement just 2
-                    // With this implementation, the original content in the remaining lines are retained
-                    .unwrap_or(span.end.line);
-
-                let span_line = if replacement_lines.peek().is_some() {
-                    // get the length of the line in the original content
+
+                let bandaid = if let Some(line) = span_lines.next() {
+                    // Replacement covers a line in original content
+
+                    // get the length of the current line in the original content
                     let end_of_line: Vec<usize> = chunk
```

Comment:
This would inaccurate for i.e.:

```rust
#[doc = r#"0"#]
#[doc = r#"a
c
b"#]
#[doc = r#"inf"#]
```

Reply:
That thing is entirely wrong I know. I'm on my way implementing a proper solution.